sdk/move-error-explain/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//! A crate for looking up the human-readable explanation of a Diem Move transaction
+//! abort code.
+//!
+//! This crate mostly exists as a temporary hack until we figure out a more
+//! sustainable approach towards publishing the Diem move framework release to
+//! crates.io.
+//!
+//! Note that the ~13 KiB error descriptions will be inlined into the final binary.
+
+#![forbid(unsafe_code)]
+
+mod errmap;
+
+pub use crate::errmap::*;
+
+use move_core_types::language_storage::ModuleId;
+use once_cell::sync::Lazy;
+
+static RELEASE_ERRMAP_BYTES: &[u8] =
+    include_bytes!("../../../language/diem-framework/releases/artifacts/current/error_description/error_description.errmap");
```

Comment:
So cargo doesn't like it when you try to publish a crate that accesses things outside of it. Lets create an errmap file and symlink it to here and then include the bytes from the file that is "local" to where this lives.

Reply:
I was wondering about that. I'll add another symlink 👍

Reply:
symlink all the things!

Reply:
:ultrafastparrot: